README.md

```diff
@@ -6,12 +6,6 @@ wrapping standalone Wayland compositors into a set of systemd units.
 WIP(ish). The main structure of subcommands and features is more or less settled and will likely
 not receive any drastic changes unless some illuminative idea comes by.
 Nonetheless, keep an eye for commits with `[Breaking]` messages.
```

Comment:
Could you also please change this, now that we are changing style of commits?

Reply:
![image](https://github.com/Vladimir-csp/uwsm/assets/920910/48fd095a-5266-4b05-827d-284540cff573)
Here's a preview of the README changes.
I've never used these icons, but they look pretty cool IMO.
https://github.com/orgs/community/discussions/16925